src/Lean/Data/Json/FromToJson.lean

```diff
@@ -63,7 +68,7 @@ def toStructured? [ToJson α] (v : α) : Option Structured :=
   fromJson? (toJson v)
 
 def getObjValAs? (j : Json) (α : Type u) [FromJson α] (k : String) : Option α :=
-  (j.getObjVal? k).bind fromJson?
+  fromJson? <| j.getObjValD k
```

Comment:
Going by https://github.com/Vtec234/lean4/commit/4c2e50d9e886f09cc36193a992e567de14278344, I decided to make the decoding even more permissible here and above, which let me implement everything without special cases for `Option _` fields.